src/common/local_file_system.cpp

```diff
@@ -1337,4 +1355,25 @@ unique_ptr<FileSystem> FileSystem::CreateLocal() {
 	return make_uniq<LocalFileSystem>();
 }
 
+#if defined(__DARWIN__) || defined(__APPLE__) || defined(__OpenBSD__)
+void LocalFileSystem::CopyFile(const string &source, const string &target, unique_ptr<FileHandle>& src_handle, unique_ptr<FileHandle>& dst_handle) {
+  int src_fd = src_handle->Cast<UnixFileHandle>().fd;
+  fclonefileat(src_fd, AT_FDCWD, target.c_str(), 0);
+}
+#elif _WIN32
+void LocalFileSystem::CopyFile(const string &source, const string &target, unique_ptr<FileHandle>& src_handle, unique_ptr<FileHandle>& dst_handle) {
+  throw NotImplementedException("CopyFile Unsupported");
```

Comment:
Would someone using Windows for development run into this?  If so I think we should implement it before merging.

Reply:
Windows version implemented at line 1099. Removed this code.